lib/bookingsync/api/client.rb

```diff
@@ -209,7 +209,8 @@ def paginate(path, options = {}, &block)
           response = call(request_method, path, options)
         end
         @pagination_first_response = response
-        data = response.resources.dup
+        data = []
+        data.concat(response.resources)
```

Comment:
Ok I have run it for 100 million iterations :smile: 
```
                                  user     system      total        real
dup                          33.540000 323.650000 357.190000 (359.187487)
nodup                        30.850000 312.870000 343.720000 (344.817036)
```
Its not ground-breaking, but looks a bit faster, lets do it.